### ISSUE TITLE
Panic with `--tmux` when tmux is already running

### ISSUE THREAD

**Issue by 9999years** [topgrade-rs/topgrade#117]
```ShellSession
$ cargo run -- --tmux
create window failed: index 0 in use
thread 'main' panicked at 'Error running Topgrade in tmux: exit status: 1', src/steps/tmux.rs:95:35
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

On latest `master` (a7c22625378a13a318624da3817c740a395e2736)

**Comment by 9999years** [comment]
Playing around with a patch for this but not getting great results.

**Comment by DottoDev** [comment]
Can I asign you to this Issue?

**Comment by 9999years** [comment]
Fixed in #165
